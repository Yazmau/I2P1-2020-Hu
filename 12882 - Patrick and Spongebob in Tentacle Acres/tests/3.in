1 24
19 89 82 22 21 57 93 49 81 5 90 56 54 13 97 79 72 99 27 32 95 30 32 42
80
1 1 13 17
1 1 4 17
1 1 13 14
1 1 8 16
1 1 4 15
1 1 9 22
1 1 13 19
1 1 8 18
1 1 15 15
1 1 20 23
1 1 4 4
1 1 24 24
1 1 13 21
1 1 4 24
1 1 10 16
1 1 17 17
1 1 14 21
1 1 9 21
1 1 24 24
1 1 11 17
1 1 1 6
1 1 10 17
1 1 4 15
1 1 7 19
1 1 8 24
1 1 1 23
1 1 23 23
1 1 3 6
1 1 2 18
1 1 21 21
1 1 10 11
1 1 17 23
1 1 23 23
1 1 14 16
1 1 3 12
1 1 15 19
1 1 23 23
1 1 13 24
1 1 12 16
1 1 23 23
1 1 11 18
1 1 24 24
1 1 4 13
1 1 16 16
1 1 17 19
1 1 8 24
1 1 3 17
1 1 17 22
1 1 18 21
1 1 13 17
1 1 24 24
1 1 12 21
1 1 14 15
1 1 16 24
1 1 21 23
1 1 8 23
1 1 14 23
1 1 7 19
1 1 1 16
1 1 5 7
1 1 24 24
1 1 20 23
1 1 12 21
1 1 3 8
1 1 6 7
1 1 21 24
1 1 14 24
1 1 2 4
1 1 14 22
1 1 12 20
1 1 7 19
1 1 10 13
1 1 10 19
1 1 7 23
1 1 24 24
1 1 20 21
1 1 1 24
1 1 24 24
1 1 7 13
1 1 14 16
