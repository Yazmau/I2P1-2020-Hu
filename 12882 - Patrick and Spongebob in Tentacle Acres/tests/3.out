97
97
54
97
97
99
99
99
97
95
22
42
99
99
97
72
99
99
42
97
89
97
97
99
99
99
32
82
99
95
90
99
32
97
93
99
32
99
97
32
99
42
93
79
99
99
97
99
99
97
42
99
97
99
95
99
99
99
97
93
42
95
99
93
93
95
99
89
99
99
99
90
99
99
42
95
99
42
93
97
